### DISCUSSION TITLE
Deploy và cấu hình website trên namecheap trong vòng 1 nốt nhạc [Part 1] | Leo's blog

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#5] [Announcements]
# Deploy và cấu hình website trên namecheap trong vòng 1 nốt nhạc [Part 1] | Leo's blog

Trước giờ mình luôn muốn sở hữu 1 site cá nhân, nhưng vì mình không có thời gian nên đợt này nghỉ việc, ở nhà cách ly mới tìm hiểu để làm được...

https://www.leohuynh.dev/blog/deploy-and-config-website-on-name-cheap-part-1

**Comment by hta218** [comment]
Looking good!
